README.md

```diff
@@ -83,8 +83,34 @@ $ yarn test
 
 :pushpin: Don't forget to update / create new tests when you contribute to code to keep the app the consistent.
 
+___
+# Services
+
+## expiration:
+  This service consists in checking every day between 1pm & 3pm if qualified documents are about to expire <br/>(_expiry date_ - _offset of document type_)
```

Comment:
c'est normal le `<br/>(_expiry date_ - _offset of document type_)` ?

Reply:
Oui, histoire de mise en page